src/MechanismGeometries.jl

```diff
@@ -26,7 +27,11 @@ abstract type AbstractGeometrySource end
 
 function visual_elements(mechanism, source::AbstractGeometrySource) end
 
-const GeometryLike = Union{AbstractGeometry, AbstractMesh}
+struct MeshFile
+    filename::String
+end
+
+const GeometryLike = Union{AbstractGeometry, AbstractMesh, MeshFile}
```

Comment:
This caused some confusion for me in MeshCatMechanisms. MeshCat also has a `GeometryLike` that used to be the same, but now additionally has `MeshFileGeometry`. This makes it even more different. Should we rename?

Reply:
Agreed that it's a bit awkward, but I don't think it's a huge problem. Honestly, unless it's causing problems I wouldn't worry about it right now.